autorandr.py

```diff
@@ -176,6 +185,7 @@ class XrandrOutput(object):
             CRTC:\s*(?P<crtc>[0-9]) |                                                   # CRTC value
             Transform: (?P<transform>(?:[\-0-9\. ]+\s+){3}) |                           # Transformation matrix
             EDID: (?P<edid>\s*?(?:\\n\\t\\t[0-9a-f]+)+) |                               # EDID of the output
+            %s |                                                                        # Properties to include in the profile
```

Comment:
I think in this particular case I'd prefer `""" + foo + """`. Yes, that's slow in Python, but this is a giant string, and imho readability of the overall thing counts here.